### DISCUSSION TITLE
Androidpag播放失败-当手机开发者选项设置不播放动画或者辅助功能里开启移除动画

### DISCUSSION THREAD

**Discussion by outlie** [Tencent/libpag#1632] [Q&A] [answered]
如标题所示，Android设备，当手机开发者选项设置不播放动画或者辅助功能里开启移除动画，pag无法播放
现象要么是全部隐藏， 有的pag文件只能播放其中一帧

**Comment by kevingpqi123** [comment, marked as answer]
这不是正常的吗，都设置了动画伸缩关闭，就是不需要动画了哈，这里影响的是系统 ValueAnimator，凡是依赖了 ValueAnimator 的都会受影响

**Reply by outlie** [reply]
这样啊， 谢谢
